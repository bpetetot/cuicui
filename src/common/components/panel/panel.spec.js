/* eslint-disable react/jsx-filename-extension */
/* eslint-env jest */
import snap from 'snap'

import Panel from './panel'

const snapshot = props => snap(Panel)({ ...props })

describe('panel', () => {
  it('should add title', snapshot({ title: 'hello world' }))
  it('should add children', snapshot({ children: 'hello world' }))
  it('should add custom className', snapshot({ className: 'custom' }))
  it('should add custom style', snapshot({ style: { color: 'red' } }))
})
