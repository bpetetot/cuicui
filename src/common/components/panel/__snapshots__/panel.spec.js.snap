// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panel should add children 1`] = `
<div
  className="cc-panel"
  style={undefined}
>
  <div
    className="cc-panel--title"
  />
  hello world
</div>
`;

exports[`panel should add custom className 1`] = `
<div
  className="cc-panel custom"
  style={undefined}
>
  <div
    className="cc-panel--title"
  />
</div>
`;

exports[`panel should add custom style 1`] = `
<div
  className="cc-panel"
  style={
    Object {
      "color": "red",
    }
  }
>
  <div
    className="cc-panel--title"
  />
</div>
`;

exports[`panel should add title 1`] = `
<div
  className="cc-panel"
  style={undefined}
>
  <div
    className="cc-panel--title"
  >
    hello world
  </div>
</div>
`;
